### DISCUSSION TITLE
add home button to quickly move to homepage

### DISCUSSION THREAD

**Discussion by MondayNitro** [Floorp-Projects/Floorp#802] [Feature Request]
please add a home button adjacent to the address bar, like in edge and chrome, thats the only feature that im missing in floorp

![Screenshot 2024-01-27 052133](https://github.com/Floorp-Projects/Floorp/assets/87489540/4e6d88b9-2fee-48da-98e7-ec38eb8c7c2e)

**Comment by CutterKnife** [comment]
https://support.mozilla.org/en-US/kb/customize-firefox-controls-buttons-and-toolbars
https://imgur.com/a/W7FRxcd

**Reply by MondayNitro** [reply]
thank you for helping
